lib/performance/EmittedAssetSizeLimitPlugin.js

```diff
@@ -0,0 +1,179 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Sean Larkin @thelarkinn
+*/
+var path = require('path');
+
+var EntrypointsOverSizeLimitWarning = require('./EntrypointsOverSizeLimitWarning');
+var AssetsOverSizeLimitWarning = require('./AssetsOverSizeLimitWarning');
+
+function EmittedAssetSizeLimitPlugin(performanceOptions) {
+	this.maxAssetSize = performanceOptions.maxAssetSize;
+	this.maxInitialSize = performanceOptions.maxInitialChunkSize;
+	this.hints = performanceOptions.hints;
+	this.errorOnHint = performanceOptions.errorOnHint;
+}
+
+module.exports = EmittedAssetSizeLimitPlugin;
+
+function formatSize(size) {
```

Comment:
The reason why I have this pulled in is for the purpose of giving formatted values for the errors/warnings. I didn't want to implement this all warnings (unless I create a proto method on each warning).